csi.proto

```diff
@@ -293,13 +293,27 @@ message DeleteVolumeRequest {
   Version version = 1;
 
   // The handle of the volume to be deprovisioned.
-  // This field is REQUIRED.
-  VolumeHandle volume_handle = 2;
+  // Either `name` or `volume_handle` SHALL be specified by the CO.
+  // A CO MAY specify the `name` when the `volume_handle` is unknown.
+  // For example, in the case where a `CreateVolume` RPC fails and the
+  // CO is unsure of the completion state of the request AND no longer
+  // needs the possibly-created volume. Alternatively, a CO MAY
+  // specify the `volume_handle` field when such information is
+  // available to it. This field is REQUIRED.
+  oneof handle {
```

Comment:
I'm not crazy about `handle` as the name for this oneof, but I can't think of anything better. using `reference` yields a weird golang API code. Naming .. ugh.